app/api/gist/index.js

```diff
@@ -0,0 +1,67 @@
+"use strict";
+
+var Promise = require("bluebird");
```

Comment:
For future proofing should we use something other than Promise. Promise hopefully will become a built in in ES6.

https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Promise